### PR TITLE
Feature/remove cache folder from s3

### PR DESCRIPTION
**Pull request by pkfec**
## Summary (required)

Create a celery schedule task to delete the folders unders cached-calls from s3 bucket.
Schedule task run every day @6a.

- Addresses # [_Issue number_]

fecgov/openFEC#3009

_Include a summary of proposed changes._

## How to test the changes locally

- redis-server, celery beat, celery worker should be UP and running in your local.

## Impacted areas of the application
List general components of the application that this PR will affect:

-  folders under cached-calls  on s3 bucket should be deleted.

## Related PRs
None

### CODE REVIEW

**Comment by pkfec** [comment]
@rjayasekera and @ccostino   Please review.

File: webservices/tasks/__init__.py [hunk 1]
```diff
@@ -22,6 +22,10 @@
             'task': 'webservices.tasks.legal_docs.refresh',
             'schedule': crontab(minute='*/5'),
         },
+        'delete_cached_call_folder': {
+            'task': 'webservices.tasks.cache_request.delete_cached_calls_folder',
+            'schedule': crontab(minute=0, hour=10),
```
Comment (lbeaufort) [inline webservices/tasks/__init__.py]: @pkfec should this be `delete_cache_calls_from_s3`?

Comment (pkfec) [reply webservices/tasks/__init__.py]: The folder name on S3 is "cached-calls". Thought of giving the same name in my function too

Comment (lbeaufort) [reply webservices/tasks/__init__.py]: The function below https://github.com/fecgov/openFEC/pull/3024/files#diff-8c5fcdb267045bc27a9bfa4825bd19e3R43 has no "d" `delete_cache_calls_folder`

**Pull request merged**